polynote-kernel/src/main/scala/polynote/kernel/lang/python/PythonInterpreter.scala

```diff
@@ -31,6 +31,11 @@ class PythonInterpreter(val kernelContext: KernelContext) extends LanguageInterp
 
   override def init(): IO[Unit] = withJep {
     jep.set("__kernel_ref", polynote.runtime.Runtime)
+
+    // Since Scala uses getter methods instead of class fields (i.e. you'd need `kernel.display().html()` instead of
+    // `kernel.display.html`), we do a little bit of acrobatics here – we create an object in Python that wraps the
+    // Runtime, and dynamically set an attribute `display` on it with Jep so that it's accessible as an attribute
+    // rather than a method.
```

Comment:
Added explanation back in, and moved the related stuff up to be with the `KernelProxy` definition.